### PR TITLE
Expose notification service

### PR DESCRIPTION
**Pull request by Novkirishki**

### CODE REVIEW

**Review by annix8 — APPROVED** [review]

**Review by belchevgb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -165,6 +165,10 @@ Take a look at the following overview of the Admin App extension samples we prov
 
 * [Search](./search/README.md) - By default when search is executed the AdminApp searches only by a content type's default field. This may not be enough, therefore you have the ability to define the fields to be used when search is performed per content type.
 
+* Custom notifications - Often when the user executes an action he has to be notified somehow about the result of his action. In such cases you have the ability to publish custom notifications in the application. This can be achieved by using the **NotificationService** and its **publishBasicNotification** method. The method accepts an object of type **NotificationInfo** that contains some configuration options about the notification's message, look, visibility duration, placement, etc. The [**notificaton command extension**](./commands-extender/notification.command.ts) provides an example how to create your own custom notifictions by using the **NotificationService**.
```
Comment (jmeleshko) [inline README.md:168]: notificaton -> notification

Comment (jmeleshko) [reply README.md:168]: notifictions -> notifications

**Review by jmeleshko — APPROVED** [review]

File: src/commands-extender/notification.command.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     * This method gets invoked when the notification action button is clicked
+     * @param context The context that contains the data item and the custom properties from the CommandProvider.
+     */
+    execute(context: ExecutionContext): Observable<any> {
+        // get the data item from the context.
+        const dataItem: DataItem = context.data.dataItem;
+
+        // publish a notification
+        const notificationInfo: NotificationInfo = {
+            message: `Notification command executed on item: ${dataItem.title}`,
```
Comment (mstratiev) [inline src/commands-extender/notification.command.ts:24]: dataItem.title? is that what we get here? Because normally it is dataItem.data.Title

Comment (Novkirishki) [reply src/commands-extender/notification.command.ts:24]: DataItem has a title property which gives us the default short text field value. It is equal to dataItem.data.Title when "Title" is the main short text field

**Pull request merged**